mz_compat.c

```diff
@@ -1093,12 +1093,19 @@ int unzGoToNextFile(unzFile file) {
     return err;
 }
 
-int unzLocateFile(unzFile file, const char *filename, unzFileNameComparer filename_compare_func) {
+#ifdef WIN32
+  #define UNZ_DEFAULT_IGNORE_CASE 1
```

Comment:
Sorry, I should have clarified, in this project I use `#  define` when indented.

Reply:
Oops. Fixed.